Sources/VLCMovieViewController.m

```diff
@@ -1649,4 +1673,36 @@ - (void)handleExternalScreenDidDisconnect:(NSNotification *)notification
     [self hideFromExternalDisplay];
 }
 
+#pragma mark - Renderers
+
+- (void)playingExternallyViewWithRendererName:(NSString *)name
+{
+    // TODO: Have a better/prettier transition between local playback frame and playingExternallyView
+    _playingExternallyView.hidden = NO;
+    _playingExternallyTitle.text = NSLocalizedString(@"PLAYING_EXTERNALLY_TITLE_CHROMECAST", nil);
+    _playingExternallyTitle.text = @"Chromecast connected";
+    _playingExternallyDescription.text = name;
+}
+
+- (void)setupCastWithCurrentRenderer
+{
+    VLCRendererItem *renderer = _vpc.renderer;
+
+    if (renderer != nil) {
+        [self playingExternallyViewWithRendererName:renderer.name];
+    } else {
+        _playingExternallyView.hidden = YES;
+    }
+    [_vpc mediaPlayerSetRenderer:renderer];
+}
+
+- (void)setupRendererDiscovererManager
+{
+    // Create a renderer button for VLCMovieViewController
+    _rendererButtton = [VLCRendererDiscovererManager.sharedInstance setupRendererButton];
+    [VLCRendererDiscovererManager.sharedInstance addSelectionHandlerWithSelectionHandler:^(VLCRendererItem * _Nonnull item) {
```

Comment:
Does this result in a strong reference to the MovieViewController ? if so that would create a memoryleak